recipes/mongos.rb

```diff
@@ -19,9 +21,7 @@
 # limitations under the License.
 #
 
-node.set['mongodb']['is_mongos'] = true
-node.set['mongodb']['shard_name'] = node['mongodb']['shard_name']
-node.override['mongodb']['instance_name'] = 'mongos'
+node.override['mongodb']['is_mongos'] = true
```

Comment:
wait, why does this need to be an override? btw, this pattern has been terrible since forever.